hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/PartitionServiceProxy.java

```diff
@@ -72,6 +73,16 @@ public boolean removeMigrationListener(String registrationId) {
     }
 
     @Override
+    public String addPartitionLostListener(PartitionLostListener partitionLostListener) {
```

Comment:
Isn't it unexpected you can register a map-partition-lost-listener from client side but not regular partition-lost-listener?